make/conf/module-loader-map.conf

```diff
@@ -94,3 +94,7 @@ PLATFORM_MODULES= \
 PLATFORM_MODULES_windows= \
     jdk.crypto.mscapi \
     #
+
+# Hook to include the corresponding custom file, if present.
+$(eval $(call IncludeCustomExtension, conf/module-loader-map.conf)) 
+
```

Comment:
Using IncludeCustomExtension is a good idea, but you should to this where module-loader-map.conf is included in common/Modules.gmk, not here. The `*.conf` files are supposed to be formatted as simple configuration files, and not include special make commands.

Reply:
Ah ok thanks Magnus, didn't realize that, I will update

Reply:
@magicus Is there an ordering issue in common/Modules.gmk? It also invokes IncludeCustomExtension but this is done before it includes the conf file.

Reply:
Yes, that's what JDK-8258411 inadvertently changed and I am fixing. I am going to now move the common/Modules.gmk IncludeCustomExtension after the module-loader-map.conf  include.

Reply:
@AlanBateman I'm not sure. Frankly, most of the IncludeCustomExtension hooks are no longer used by Oracle since we've moved most of our code out in the open, so they kind of bit rot... I should probably do a sweep and ask the community which one to still keep, and remove the rest.

Reply:
> @AlanBateman Yes, that's what JDK-8258411 inadvertently changed and I am fixing. I am going to now move the common/Modules.gmk IncludeCustomExtension after the module-loader-map.conf include.

Andrew, why can't you just use the existing IncludeCustomExtension then to load your own module-loader-map.conf? Or maybe that's just what you are going to do?

Reply:
The previous behavior provided the ability to "extend" the upstream MODULE lists, ie.just add OpenJ9 modules to the base module lists, see: https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/01e13c398721628540babac94ac8663be716c0a8/closed/custom/common/Modules.gmk#L21

The present https://github.com/openjdk/jdk/blob/master/make/conf/module-loader-map.conf will simply over-write whatever is set in the BOOT_MODULES list.

What I am now going to do is move the IncludeCustomExtension after the module-loader-map.conf  inclusion.
The other alternative would be to change module-loader-map.conf so it appended to any existing BOOT_MODULES variable, but I think you suggested that file is a straight conf file?

@magicus thoughts?

Reply:
Note, it was the change from "appending" to BOOT_MODULES to just "assigning" that broke the existing behaviour with the JDK-8258411 change.

Reply:
@andrew-m-leonard (Seems I can't get github to tag you???)

That sounds good. I think you could move the IncludeCustomExtension to after all *.conf files, to future-proof it and to make it a bit more consistent -- "first include conf files, then adjust them in custom extensions".

Reply:
@magicus yes, that's exactly the new commit i've just pushed.

@(lookups) have been a bit wobbly recently with github.com I have noticed, I can find some people but not others!

Reply:
@andrew-m-leonard try searching at
https://github.com/orgs/openjdk/people

Reply:
@magicus @AlanBateman please see new commit (webrev https://webrevs.openjdk.java.net/?repo=jdk&pr=2219&range=01)